lib/v2/sehuatang/router.js

```diff
@@ -1,4 +1,5 @@
 module.exports = function (router) {
+    router.get('/user/:uid', require('./user'));
     router.get('/bt/:subforumid?', require('./index'));
     router.get('/picture/:subforumid', require('./index'));
```

Comment:
```suggestion
    router.get('/bt/:subforumid?', require('./index'));
    router.get('/picture/:subforumid', require('./index'));
    router.get('/user/:uid', require('./user'));
```
Please add new entries in alphabetical order.
docs: https://docs.rsshub.app/joinus/script-standard.html